butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/conditions/java/JavaMatch.java

```diff
@@ -106,6 +132,15 @@ protected TUExecutionResult execution(File transformedAppFolder, TransformationC
             fileInputStream = new FileInputStream(javaClassFile);
             CompilationUnit compilationUnit = JavaParser.parse(fileInputStream);
 
+            compilationUnit.setStorage(javaClassFile.toPath());
```

Comment:
Since we have already a hold of the file (`javaClassFile`), do we really need to manipulate `compilationUnit` to check if file is named `package-info.java`? What if we just do `javaClassFile.getName()`?

In the future, if JavaParser adds `isPackageInfo` then it would make sense to delegate that check to JavaParser, but not at this time.

Does that make sense?